pkg/remoteclient/remoteclient.go

```diff
@@ -1,6 +1,6 @@
 package remoteclient
 
-//go:generate mockgen -source=./remoteclient.go -destination=./mock/remoteclient_generated.go -package=mock
+//go:generate mockgen -destination=./mock/remoteclient_generated.go -package=mock  github.com/openshift/hive/pkg/$GOPACKAGE Builder
```

Comment:
Had to switch from source mode to reflect mode to be able to select a specific interface.

Reply:
Switched back to the source mode and moved dialer related code into separate files as per Eric's suggestion.